### DISCUSSION TITLE
Support sameSiteCookieValue

### DISCUSSION THREAD

**Discussion by mambose** [solspace/craft-freeform#469] [Feature Requests]
see https://craftcms.com/docs/4.x/config/general.html#samesitecookievalue

https://github.com/solspace/craft-freeform/blob/306f27eeb9bedaad49580c6a4a8b6129e869879e/packages/plugin/src/Bundles/Form/Tracking/Cookies.php#L32

Otherwise the Owasp ZAP scanner brings up the following alert:

> A cookie has been set without the SameSite attribute, which means that the cookie can be sent as a result of a 'cross-site' request. The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion, and timing attacks.
> 
> Ensure that the SameSite attribute is set to either 'lax' or ideally 'strict' for all cookies.

**Comment by kjmartens** [comment]
Thanks @mambose. We'll check into this and see if we can implement that soon. 🙂

**Comment by kjmartens** [comment]
This has now been implemented in Freeform 3.13.22.1+ and 4.0.9+. 🙂
